components/PickImage.js

```diff
@@ -1,14 +1,12 @@
 import React, {useState, useEffect} from 'react'
-import { StyleSheet, Text, View, Image } from 'react-native'
-import Constants from 'expo-constants'
+import { Text, View, Image } from 'react-native'
 import * as ImagePicker from 'expo-image-picker'
-import * as Haptics from 'expo-haptics';
-
+import * as Haptics from 'expo-haptics'
 //Libraries
 import { Button } from "react-native-paper"
+import styles from "./PickImageStyles"
```

Comment:
It wont let me fix the merge conflicts I think youll have to do this on your end.